linode/helper/compare.go

```
@@ -10,6 +10,12 @@ import (
 )
 
 func CompareTimeStrings(t1, t2, timeFormat string) bool {
+	// time.Parse(...) does not support parsing empty string times,
+	// but this is a valid case in certain scenarios.
+	if t1 == "" && t2 == "" {
+		return true
+	}
+
```

Can the time string become empty strings? I feel we should return an error when they are empty strings..

---

After discussion, we will fix this in a following up PR.